.github/workflows/ci.yml

```diff
@@ -30,8 +30,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: ["3.1", "3.2"]
-        rails: ["6.1", "7.0"]
+        ruby: ["3.2", "3.3"]
+        rails: ["6.1", "7.0", "7.1", "7.2", "8.0"]
```

Comment:
Looks like we can probably drop 6.1 and 7.0. I don't mind dropping 7.0 a few months early personally.

https://endoflife.date/rails